tests/tbasic

```diff
@@ -138,4 +138,12 @@ req -new -batch -key "${PRIURI}" -out ${TMPPDIR}/rsa_csr.pem'
 ossl '
 req -in ${TMPPDIR}/rsa_csr.pem -verify -noout'
 
+title PARA "Test fetching public keys without PIN"
+ORIG_OPENSSL_CONF=${OPENSSL_CONF}
+sed "s/^pkcs11-module-token-pin.*$/##nopin/" ${OPENSSL_CONF} > ${OPENSSL_CONF}.nopin
+OPENSSL_CONF=${OPENSSL_CONF}.nopin
+ossl 'pkey -in $BASEURI -pubin -pubout -out ${TSTOUT}.pub.nopin'
```

Comment:
In the original test I had also test with EC keys. Not a big deal and should not matter much, but would be good to have better coverage.

https://github.com/latchset/pkcs11-provider/compare/main...Jakuje:login

Reply:
Alright, I can add also the EC one